bin/codacy-analysis-cli.sh

```diff
@@ -78,41 +78,45 @@ analysis_file() {
   fi
 }
 
-output_file() {
-  local filename="";
+change_output_file() {
   local is_filename=0;
+  local new_command="";
   for arg; do
     case "$arg" in
       -*)
         case ${arg} in
           --output)
             is_filename=1 # next argument will be the file
+            new_command="${new_command} ${arg}"
             ;;
         esac
         ;;
       *)
         if [ ${is_filename} -eq 1 ]; then
-          if [ -n "$filename" ]; then
+          if [ -n "$OUTPUT" ]; then
             echo "Please provide only one output file" >&2
             exit 1
           else
             is_filename=0
-            filename="$arg"
+            OUTPUT_DIRECTORY=$(cd $(dirname "${arg}") && pwd -P)
+            OUTPUT_FILENAME=$(basename "${arg}")
+            OUTPUT="${OUTPUT_DIRECTORY}/${OUTPUT_FILENAME}"
```

Comment:
This one might not work with local since its needed function wide. Check this case.